packages/logo/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@heycar-uikit/logo",
+  "version": "1.0.0",
+  "description": "HeyCar-UIKit logo",
+  "main": "dist/index.js",
+  "module": "./dist/esm/index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Just out of curiosity , why do we have this test script saying no test specified ? in a package that actually has tests inside. This is not the only component with this line in its package.json. Some components have this line some don't . any reason ? @derek-heycar I also wonder your opinion on this.

Reply:
I think we can actually remove this as we have specified jest for the main package json